### ISSUE TITLE
[Feature Request] Add Mercure

### ISSUE THREAD

**Issue by myavchik** [spiral/roadrunner#180]
Add [Mercure](https://github.com/dunglas/mercure) or something similar.

But first, wait until https://github.com/dunglas/mercure/issues/105 solved.

**Comment by wolfy-j** [comment]
This looks cool, it must be pretty easy to integrate since it's Go based.

**Comment by wolfy-j** [comment]
What about Web-Socket broadcasting we've added last week? https://github.com/spiral/broadcast

**Comment by myavchik** [comment]
Very good module, but it have a little bit different uses.
WebSockets are not battery-efficient way to emit "changes".

I think "Broadcast messages over web-sockets" title should be renamed. WebSockets uses TCP, which is point to point, and provides no broadcast support.

**Comment by wolfy-j** [comment]
Correct, but there is also broken behind the hood which provides the ability to broadcast via multiple instances. But I see the point about push notifications.
